components/Form/FormInput.tsx

```diff
@@ -1,11 +1,25 @@
 import classNames from 'classnames';
-import { useState } from 'react';
+import React, { FC, useState } from 'react';
 
-import { FormLabel } from './FormLabel';
+import { FormLabel } from './FormLabel.tsx';
 
 const regex = /^[a-zA-Zá-žÁ-Ž0-9 .]+$/;
 
-export const FormInput = ({
+type Props = {
+  type: 'email' | 'number' | 'text' | 'password' | undefined;
+  defaultValue?: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  labelClassName?: string;
+  label: string;
+  required?: boolean;
+  value?: any;
+  pattern?: string;
+  className?: any;
```

Comment:
Avoid using `any`, especially when it is THIS easy. I mean className is a string

![CleanShot 2022-06-22 at 21 35 54](https://user-images.githubusercontent.com/12488648/175121622-bfd03892-8eee-4143-8d6a-7f50c71430d8.png)

Reply:
um, I dont know why I used the any at className. Error.